rancher2/0_provider_upgrade_test.go

```diff
@@ -435,15 +435,14 @@ func testAccRancher2UpgradeVars() resource.TestCheckFunc {
 			rancherTokenKey := os.Getenv("RANCHER_TOKEN_KEY")
 			rancherAdminPass := os.Getenv("RANCHER_ADMIN_PASS")
 			if err := os.Setenv("RANCHER_TOKEN_KEY", token); err != nil {
-				fmt.Printf("Failed to update RANCHER_TOKEN_KEY on ressource %s with err %s", k, err.Error())
+				fmt.Printf("Failed to update RANCHER_TOKEN_KEY on resource %s with err: %s", k, err.Error())
```

Comment:
Based on our discussion I'd leave these os.Env and print statement in but comment in the code that they fix a race condition in Drone. Otherwise, you'd have to run Drone 8-10 times to get a passing build. Add what you fixed into the PR description as well and reference it here https://github.com/rancher/rancher/issues/42212. Thank you!